docs/source/sci_api_tutorial/mutate_stru.rst

```diff
@@ -0,0 +1,189 @@
+==============================================
+Mutate Structure
+==============================================
+
+Briefs
+==============================================
+The target for this tutorial to mutate the target protein at defined residue(s). The main function provided are ``assign_mutant()`` and ``mutate_stru()``, in which ``mutate_stru()`` solves the problem of protein structural prediction upon mutation. It means to determine which mutation to address and determine the structure of the mutant of the source protein caused by residue substitution, deletion, and insertion. (see also: structure_prediction module for an alternative solution)
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/mutation_mutate_stru_io.svg
```

Comment:
same as the comment in assign_mutant. Make the color changes. (green->blue, blue->gray in this case)